crates/viewer/re_viewer_context/src/gpu_bridge/image_to_gpu.rs

```diff
@@ -319,6 +319,10 @@ pub fn texture_creation_desc_from_color_image<'a>(
         let datatype = image.format.datatype();
 
         match (color_model, datatype) {
+            (ColorModel::L, ChannelDatatype::U8) => (
+                cast_slice_to_cow(&image.buffer),
+                SourceImageDataFormat::WgpuCompatible(TextureFormat::R8Unorm),
```

Comment:
Without this change this was being sent to the shader as a `R8Uint` which caused a wgpu validation error

Reply:
ah nice, I thiiiink this is the right place & right fix given how fallbacks work. But since lack some test coverage in this area I'd like you to have a look what happens before after for a few different scenarios - I've put those as checkboxes in the ci description so they get picked up by "checkbox ci"

There should be respective snippets for all of those - should hopefully be just a matter of running existing rust/python snippets and maybe adjusting a thing. Reach out if this ends up being substantial work, thx!